doc/source/plugins.rst

```diff
@@ -359,3 +359,20 @@ Settings:
 The S3 connection options are handled by the
 `fsspec <https://filesystem-spec.readthedocs.io/en/latest/features.html#configuration>`_
 configuration mechanism.
+
+Caching remotely read data
+**************************
+
+The ``use_fsspec_cache`` plugin can be used to turn on cachcing for remotely read files. See below for
+cache cleaning.
+
+Settings used from the product list:
+  - ``fsspec_cache`` - name of the cache implementation. Can be one of ``blockcache``, ``filecache`` or
+    ``simplecache``
+  - ``fsspec_cache_dir`` - location where the caching is done. If it does not exist, it will be created
```

Comment:
I think it would be nicer to have the two config options under the same section, and maybe also a way to pass any args to the cache, eg:
```yaml
fsspec_cache:
  type: filecache
  cache_args: <fsspec cache arguments>
```
and refer to the fsspec cache doc

Reply:
Fixed in https://github.com/pytroll/trollflow2/pull/205/commits/1c2844e9249f375897490a1f85d8540413327c1c